packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -6,6 +6,7 @@ import { useState } from "react";
 
 import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { WorkflowPermissions } from "@calcom/lib/server/repository/workflow-permissions";
```

Comment:
This user has no permission to see the team's workflows, but they are still shown as "readonly":
<img width="545" height="164" alt="Screenshot 2025-08-07 at 12 33 02 PM" src="https://github.com/user-attachments/assets/b227e038-b241-4290-b1ab-273207903e5f" />

When clicking on the workflow to see the details it throws `unauthorized`